nimbus/src/lib.rs

```diff
@@ -33,7 +33,7 @@ const DB_KEY_NIMBUS_ID: &str = "nimbus-id";
 /// It should hold all the information needed to communicate a specific user's
 /// experimentation status
 pub struct NimbusClient {
-    http_client: Client,
+    http_client: Box<dyn SettingsClient + Send>,
```

Comment:
naming nit: the `http_client` here might be better called `settings_client` after this refactor, but I'm not sure how invasive a change that would be.

Reply:
I changed the `collection_name` client as I think it's still worth calling out it ignored everything in `RemoteSettingsClient` except the URL. Changing to `settings_client` wasn't at all intrusive, so 👍